svn-push

```diff
@@ -174,14 +174,14 @@ echo "Hit enter to proceed with SVN commit and Git tag"
 read OK
 
 if [ -z "$has_tag" ]; then
-	cd $git_tmp_repo_dir
+	cd "$git_root"
```

Comment:
@fjarrett silly me. It was doing a Git push from the temp Git directory to the main Git repo on our own systems, as opposed to pushing to the external remote (on GitHub).